src/main/java/core/basesyntax/animals/birds/Penguin.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.animals.birds;
+
+import core.basesyntax.animals.Bird;
+import core.basesyntax.animals.interfaces.IFish;
+
+public class Penguin extends Bird implements IFish {
```

Comment:
Penguin implements IFish is not very logical, let's call out interfaces:
swimmable 
flyable 
etc, so they represent some behaviour